elixir/apps/web/lib/web/live/resources/components.ex

```diff
@@ -57,119 +54,118 @@ defmodule Web.Resources.Components do
         {id, new_form}
       end
 
-    assigns = Map.put(assigns, :forms_by_protocol, forms_by_protocol)
+    assigns =
+      assigns
+      |> Map.put(:forms_by_protocol, forms_by_protocol)
+      |> Map.put(
+        :traffic_filters_enabled?,
+        Domain.Accounts.traffic_filters_enabled?(assigns.account)
+      )
 
     ~H"""
     <fieldset class="flex flex-col gap-2">
-      <legend class="mb-2">Traffic Restriction</legend>
-
-      <div>
-        <.input type="hidden" name={"#{@form.name}[all][protocol]"} value="all" />
-
-        <div class="items-center flex flex-row h-12">
-          <div class="flex-none w-32">
-            <.input
-              type="checkbox"
-              field={@forms_by_protocol[:all]}
-              name={"#{@form.name}[all][enabled]"}
-              checked={Map.has_key?(@forms_by_protocol, :all)}
-              value="true"
-              label="Permit All"
-            />
-          </div>
-        </div>
+      <div class="mb-1 flex items-center justify-between">
+        <legend>Traffic Restriction</legend>
+
+        <%= if @traffic_filters_enabled? == false do %>
+          <.link navigate={~p"/#{@account}/settings/billing"} class="text-sm text-primary-500">
+            <.badge type="primary" title="Feature available on a higher pricing plan">
+              UPGRADE TO UNLOCK
+            </.badge>
+          </.link>
+        <% end %>
       </div>
 
-      <div>
-        <.input type="hidden" name={"#{@form.name}[icmp][protocol]"} value="icmp" />
-
-        <div class="items-center flex flex-row h-12">
-          <div class="flex-none w-32">
-            <.input
-              type="checkbox"
-              field={@forms_by_protocol[:icmp]}
-              name={"#{@form.name}[icmp][enabled]"}
-              checked={Map.has_key?(@forms_by_protocol, :icmp)}
-              value="true"
-              disabled={Map.has_key?(@forms_by_protocol, :all)}
-              label="ICMP"
-            />
+      <p class="text-sm text-neutral-500">
+        Restrict traffic only to specified protocols and ports. By default, <strong>all</strong>
+        traffic to this Resource allowed.
```

Comment:
Here's where I think we may need to make sure we're super clear with messaging.  I think we should be careful with the word `traffic`.  Reading this sentence in a bubble, someone might get the impression that all users are allowed to access this machine by default.  Wondering if we should say something like:

```suggestion
        Restrict accessible protocols and ports. By default, <strong>all</strong>
        protocols and ports to this Resource are permitted to any user that is granted access by a Policy.
```

Reply:
Yeah I see what you mean. I'll play with the wording a little. Maybe

> Restrict access to the specific protocols and ports. By default, all protocols and ports are accessible.

Reply:
Updated!